rotkehlchen/db/schema.py

```diff
@@ -680,6 +680,10 @@
     manual_price INTEGER NOT NULL CHECK (manual_price IN (0, 1)),
     owner_address TEXT,
     blockchain TEXT GENERATED ALWAYS AS ("ETH") VIRTUAL,
+    is_lp INTEGER NOT NULL CHECK (is_lp IN (0, 1)),
+    image_url TEXT,
+    usd_price TEXT NOT NULL,
```

Comment:
is it always USD? Don't they sometimes keep other asset prices?

Reply:
This is a secondary price, the primary price is stored in columns `last_price` + `last_price_asset`. This column is needed in order to don't recalculate usd price each time nfts balances are queried from cache

Reply:
And why not? Where do you need the usd price? The frontend should take care of it no? They have the price mappings. I am still a bit confused here.